README.md

```diff
@@ -1,2 +1,76 @@
 # packaging
 Alumet docker images and distro-specific packages
+
+# Table of Contents
+- [packaging](#packaging)
+- [Table of Contents](#table-of-contents)
+- [How to install ?](#how-to-install-)
+- [How to uninstall](#how-to-uninstall)
+- [What does the RPM do ?](#what-does-the-rpm-do-)
+- [Use the reusable workflow to build RPMs](#use-the-reusable-workflow-to-build-rpms)
+  - [build\_rpm.yaml](#build_rpmyaml)
+    - [Inputs](#inputs)
+    - [Example of usage](#example-of-usage)
+
+
+When you're downloading the rpm, use a compatible version particularly if you're not on fedora.
+
+| Version of fedora 	| Version of glibc 	|
+|-------------------	|-----------------	|
+| Fedora Linux 42   	| glibc 2.40      	|
+| Fedora Linux 41   	| glibc 2.40      	|
+| Fedora Linux 40   	| glibc 2.39      	|
+| Fedora ubi 8.3   	    | glibc 2.28      	|
+| Fedora Linux 9.5   	| glibc 2.34      	|
+
+# How to install ? 
+
+```bash
+sudo rpm -i <rpm file>
+```
+
+# How to uninstall
+
+List all installed Alumet package: 
+
+```bash
+rpm -qa | grep -i alumet
+```
+
+Remove the correct Alumet package 
+```bash
+sudo rpm -e <package>
+```
+
+# What does the RPM do ? 
+
+The RPM create a folder **alumet** inside the */etc/* folder. Here will be put the **alumet-config.toml** file which is the config file for Alumet by default. 
```

Comment:
Just list the package content (there's a command for that, I don't remember which one).